draft-ietf-ppm-dap.md

```diff
@@ -1518,11 +1500,9 @@ violation. Note that this is also enforced by the Helper during the aggregation
 interaction. The Leader MAY also abort the upload interaction and alert the
 Client with error `reportRejected`.
 
-The Leader MUST ignore any report whose timestamp is before the task's
-`task_start`, or is past the end time `task_start + task_duration`. When it does
-so, it SHOULD also abort the upload interaction and alert the Client with error
-`reportRejected`. Clients MUST NOT upload a report if its timestamp would be
-earlier than `task_start` or later than `task_start + task_duration`.
+The Leader MUST ignore any report whose timestamp is outside of the task's
+`time_interval`. When it does so, it SHOULD also abort the upload interaction
+and alert the Client with error `reportRejected`.
```

Comment:
Notice that we had a bug here: DAP time intervals are half-open, which is to say that `start + duration` is excluded. So this language saying that reports should be ignored if timestamps are "past" or "later" `task_start + task_duration` was arguably wrong, or at least inconsistent with how we handle intervals elsewhere.